### PR TITLE
Update deprecated KeyEvent keyIdentifier call

### PR DESCRIPTION
**Pull request by afathman**
https://www.chromestatus.com/features/5316065118650368 keyIdentifier will no longer be supported as of Chrome 54.

### CODE REVIEW

**Comment by afathman** [comment]
I signed the agreement! 👍

**Comment by mashedcode** [comment]
@afathman I figured that you can rerun the CI test yourself by force pushing with no modification.

File: closure/goog/events/keyhandler.js [hunk 1]
```diff
@@ -429,11 +429,11 @@ goog.events.KeyHandler.prototype.handleEvent = function(e) {
       }
     }
   } else if (
-      be.keyIdentifier &&
-      be.keyIdentifier in goog.events.KeyHandler.keyIdentifier_) {
+      be.key &&
+      be.key in goog.events.KeyHandler.keyIdentifier_) {
```
Comment (mashedcode) [inline closure/goog/events/keyhandler.js]: Use clang-format. These three lines fit on one line now.

File: closure/goog/events/keyhandler.js [hunk 2]
```diff
@@ -429,11 +429,11 @@ goog.events.KeyHandler.prototype.handleEvent = function(e) {
       }
     }
   } else if (
-      be.keyIdentifier &&
-      be.keyIdentifier in goog.events.KeyHandler.keyIdentifier_) {
+      be.key &&
```
Comment (mashedcode) [inline closure/goog/events/keyhandler.js]: I'm not highly familiar with this code but this seems to cover an edge-case. Maybe the use of the deprecated `keyIdentifier` property is justified?
Please explain why you think that this is an issue.
Furthermore please add a test case for this specific edge-case and explain when this code will be reached.
Note that `keyIdentifier` is still being used in test code: https://github.com/google/closure-library/blob/8bbc453a8808aa77af91f959fb0dc97be058cd97/closure/goog/events/keyhandler_test.js#L705

**Comment by mashedcode** [comment]
Please close this PR.
